### ISSUE TITLE
Heading tags incorrect on pages with tabs, expansion panels, and cards

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2493]
Many pages with tabs, cards and expansion panels are missing and h2 or h3. 

Tab and expansion panel labels are not tagged as headings, so unless the tab or panel has a heading as part of the content, the heading structure for the page is conrrect.

Example: https://onrr.gov/references/valuation?tabs=unbundling,standardized-ucas&panel=limitations-on-the-use-of-the-unbundling-website

This skips from h2 to h4. On this page, there is an h3 below the expansion panels on the second level tabs, but those aren't nested within the h2 and h4
![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/6e6cd190-299e-407a-8469-9e36de4b9b98)

**Comment by eelzi-ONRR** [comment]
Example: https://onrr.gov/getting-started/new-reporter-checklist

This one skips from h1 (page title) to h4 (card title). The tabs and panel are not headings.
![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/2180ed9c-01ad-4b23-bef6-3b41af9642b3)

**Comment by greensomes** [comment]
@eelzi-ONRR and @Maroyafaied, the solution I am pursuing includes bumping the header levels in the CMS to the appropriate level to follow the preceding header (h4 to h3 that follows an h2, for example) and then styling those headers so the appearance remains the same. Does this sound reasonable from a content editing perspective or otherwise?

The caveat here is that there are instances in which the header levels go from h1 to h4. See the New Reporter Checklist screenshot in this issue above. In this case, the headers would have to be bumped from h4 to h2 with the styles updated to reflect how an h4 is rendered. Are we still ok with this?

**Comment by eelzi-ONRR** [comment]
@greensomes and @Maroyafaied According to the trusted tester training, "Content that is styled to look like and function like a heading must be programmatically defined as a heading" and this applies to consistency within the page. So as long as those h2s that look like h4s are consistent for all instances on that page, then this is okay.

**Comment by Maroyafaied** [comment]
moving to next sprint to come up with a plan and approach

**Comment by eelzi-ONRR** [comment]
I've inventoried all the pages and which ones have headings errors: https://doimspp-my.sharepoint.com/:x:/g/personal/elzie_mms_gov/ESba5npiehFEmhNcpwF6-qYB5Bh3Nq7aEyHnlX5x_IcFrg?e=EzLQlp

Some are easy fixes that I can fix, but many are accordion menus that have need to be coded as headings
